gpii/node_modules/eventLog/src/installID.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Retrieves an "installation ID", which is something that uniquely identifies a particular machine.
+ *
+ * The installation ID is a string that's based on the OS specific "machine ID".
+ *
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The R&D leading to these results received funding from the
+ * Department of Education - Grant H421A150005 (GPII-APCP). However,
+ * these results do not necessarily represent the policy of the
+ * Department of Education, and you should not assume endorsement by the
+ * Federal Government.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    fs = require("fs"),
+    crypto = require("crypto");
+
+fluid.defaults("gpii.installID", {
+    gradeNames: ["fluid.component", "fluid.contextAware"],
+    contextAwareness: {
+        platform: {
+            checks: {
+                test: {
+                    contextValue: "{gpii.contexts.test}",
+                    gradeNames: "gpii.installID.test",
+                    priority: "first"
+                },
+                windows: {
+                    contextValue: "{gpii.contexts.windows}",
+                    gradeNames: "gpii.installID.windows"
```

Comment:
In the Windows repo: [gpii/node_modules/windowsMetrics/src/windowsMetrics.js](https://github.com/stegru/windows/blob/4d6a1680fd65f4f7c5a4828a8156175f96e71106/gpii/node_modules/windowsMetrics/src/windowsMetrics.js#L100)

It will only be referenced under the windows context, but if it offends you I can use another technique.